### PR TITLE
*: cut 0.8.0 release

### PR DESCRIPTION
**Pull request by surajssd**
Fixes https://github.com/habitat-sh/habitat-operator/issues/355

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 # Habitat operator CHANGELOG
 
+## [v0.8.0](https://github.com/habitat-sh/habitat-operator/tree/v0.8.0) (11-9-2018)
+[Full changelog](https://github.com/habitat-sh/habitat-operator/compare/v0.7.2...v0.8.0)
+
+### Bug fixes
+
+- Fix namespace image and cluster names in CircleCI configuration [#321](https://github.com/habitat-sh/habitat-operator/pull/321)
```
Comment (krnowak) [inline CHANGELOG.md]: I'd drop it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+- Makefile's receipe `clean-test` ignores errors [#323](https://github.com/habitat-sh/habitat-operator/pull/323)
```
Comment (krnowak) [inline CHANGELOG.md]: This too.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+- Makefile: add `clean-test` as a pre-requisite to `e2e` [#329](https://github.com/habitat-sh/habitat-operator/pull/329)
```
Comment (krnowak) [inline CHANGELOG.md]: Ditto.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+- Update GKE version used to run e2e tests from `1.X.Y` to `1.X` [#333](https://github.com/habitat-sh/habitat-operator/pull/333)
```
Comment (krnowak) [inline CHANGELOG.md]: Same.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,36 @@
...
+- RBAC: Remove permission for Deployments with removal of Habitat spec v1beta1 [#339](https://github.com/habitat-sh/habitat-operator/pull/339)
+
+
+### Features & Enhancements
+
+- Added new topology label `operator.habitat.sh/topology` (see deprecation section for information about the older label) [#332](https://github.com/habitat-sh/habitat-operator/pull/332)
+- controller: change generated StatefulSet version from v1beta1 to v1 [#334](https://github.com/habitat-sh/habitat-operator/pull/334)
+- vendor kubernetes `code-generator` [#337](https://github.com/habitat-sh/habitat-operator/pull/337)
```
Comment (krnowak) [inline CHANGELOG.md]: I'd drop it.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,36 @@
...
+- Add support for Kubernetes 1.11 [#340](https://github.com/habitat-sh/habitat-operator/pull/340)
+- RBAC: Harden RBAC policies for the operator enabling operator to run in two modes, clusterwide and namespaced [#346](https://github.com/habitat-sh/habitat-operator/pull/346)
+- RBAC version: update all artifacts to rbac.authorization.k8s.io/v1 [#353](https://github.com/habitat-sh/habitat-operator/pull/353)
+- test: sync RBAC rules in various places using go test [#349](https://github.com/habitat-sh/habitat-operator/pull/349)
```
Comment (krnowak) [inline CHANGELOG.md]: This too.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,36 @@
...
+- Display version when booting operator [#350](https://github.com/habitat-sh/habitat-operator/pull/350)
+
+
+### Deprecations
+
+- Habitat CRD spec: drop support for v1beta1 [#331](https://github.com/habitat-sh/habitat-operator/pull/331)
+- Older topology label `topology` will be deprecated in two more releases [#332](https://github.com/habitat-sh/habitat-operator/pull/332)
```
Comment (krnowak) [inline CHANGELOG.md]: Topology label `topology` is deprecated and will be removed in two releases.

**Comment by surajssd** [comment]
@krnowak this is updated.

**Pull request merged**